packages/babel-plugin-strip-runtime/src/index.ts

```diff
@@ -21,23 +21,33 @@ export default declare<PluginPass>((api) => {
     },
     visitor: {
       Program: {
-        exit(path) {
-          if (!this.opts.compiledRequireExclude && this.opts.styleSheetPath) {
-            preserveLeadingComments(path);
+        exit(path, { file } ) {
+
+          if (this.opts.compiledRequireExclude) {
```

Comment:
I've reused this variable `compiledRequireExclude`.
So if it's enabled, it
- does not add  `require("${this.opts.styleSheetPath}?style=${params}");`
- add styleRules to metadata for later use